addons/compat_cup_terrains/config.cpp

```diff
@@ -13,7 +13,7 @@ class CfgPatches {
             "CAStructures_E_Ind_Ind_FuelStation",
             "CAStructures_PMC_FuelStation",
             "CUP_Buildings_Config",
-            "ace_refuel" // not a sub-component because it's all this compat does
+            "ace_common"
         };
         skipWhenMissingDependencies = 1;
```

Comment:
not a big deal but can remove
```suggestion
```

Reply:
Don't we want this compat to remain "load only when CUP is present"?

Reply:
Which do you mean: Do you want to revert the change from `ace_refuel` to `ace_common` or remove the ACE dependency altogether?